Tests/AT010.Set-CosmosDbAccountComponents.Acceptance.Tests.ps1

```diff
@@ -33,6 +33,12 @@ Describe "Set-CosmosDbAccountComponents Tests" -Tag "Acceptance-ARM" {
         $TestColl.Id | Should be $config.cosmosDbTestConfig.Databases[0].Collections[0].CollectionName
     }
 
+    It "Should create a collection with IndexingMode set to None" {
+        $TestColl = Get-CosmosDbCollection -Context $TestContext -Database $config.cosmosDbTestConfig.Databases[0].DatabaseName `
```

Comment:
Use [splatting](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_splatting?view=powershell-6#splatting-with-hash-tables) in place of backticks